pkg/apis/trainer/v1alpha1/trainjob_types.go

```diff
@@ -271,6 +271,11 @@ type PodSpecOverride struct {
 	// +listType=map
 	// +listMapKey=name
 	SchedulingGates []corev1.PodSchedulingGate `json:"schedulingGates,omitempty"`
+
+	// Override for the Pod's image pull secrets.
```

Comment:
```suggestion
	// ImagePullSecrets overrides the image pull secrets for the Pods in the target job templates.
```

Reply:
Sorry, I missed this comment earlier. Fixed.